PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/layers/receiver.h

```diff
@@ -3,14 +3,16 @@
 #include "./layersWrapper.h"
 #include "../utils/queue_wrapper.h"
 #include "flit.h/"
+#include <vector>
 
 class Receiver {
 public:
 	Globals globals;
 	LayersWrapper layers;
-	QueueWrapper<Flit> listenOn;
 	QueueWrapper<Flit> sendOn;
```

Comment:
This is supposed to be a queue of TLPs not flits, because this will be passed to the application layer